code/modules/station_goals/bluespace_tap.dm

```diff
@@ -260,7 +260,7 @@
 	. = ..()
 
 	if(length(active_nether_portals))
-		icon_state = "redspace_tap"
+		icon_state = "Cascade_tap"
```

Comment:
what is that supposed to be? 
there is no icon like that?
![image](https://github.com/ParadiseSS13/Paradise/assets/61080616/c5b5f801-bb19-49d3-bfbf-489287ecc4de)